grammars/javascript.cson

```diff
@@ -10,6 +10,7 @@
   'pjs'
   'xsjs'
   'xsjslib'
+  'jscad'
```

Comment:
Let's leave `js` at the very top since it's the most common type, but yeah I think everything else can be alphabetized.